src/ServiceProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Remember to allow Laravel to discover this provider automatically for autoregister.